style.css

```diff
@@ -0,0 +1,784 @@
+    font-size: 2vw;
+    font-weight: 600;
+    font-style: italic;
+  }
+
+  .speacker-title-mejor {
+    font-size: 1.7vw;
+    line-height: 3vw;
+  }
+
+  .speacker-divider {
+    margin-top: 0.8rem;
+    width: 10%;
+    border: 0.5px solid #d3d3d3;
+    margin-bottom: 10px;
+  }
+
+  .footer-top {
+    background-color: #2e2e2e;
+    width: 100%;
+    height: 150px;
+  }
+
+  .footer-top h2 {
+    margin-bottom: 15px;
+    margin-top: 40px;
+    text-align: center;
+    color: #fff;
+    padding-top: 13px;
+    font-size: 3.5vw;
+  }
+
+  .footer-top hr {
+    margin: 0 auto;
+    margin-top: 0.8rem;
+    width: 10%;
+    border: 0.5px solid #c23f30;
+    margin-bottom: 25px;
+  }
+
+  .footer-top img {
+    width: 50px;
+  }
+
+  .footer-bottom img {
+    width: 150px;
+  }
+
+  .footer-bottom {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    padding: 0 24px 0 24px;
+    height: 150px;
+    width: 100%;
+  }
+
+  .partner-images {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+  }
+
+  .partner-images img {
+    margin-right: 5%;
+  }
+
+  .mobile-menu-button {
+    margin: 20px 30px;
+    background-image: url("images/menu.png");
+    order: 1;
+    background-size: contain;
+    background-repeat: no-repeat;
+    width: 25px;
+    height: 25px;
+    position: fixed;
+  }
+
+  .menu-button-open {
+    margin: 20px 30px;
+    background-image: url("images/close.png");
+    order: 1;
+    background-size: contain;
+    background-repeat: no-repeat;
+    width: 25px;
+    height: 25px;
+    position: relative;
+  }
+
+  .visible-mobilemenu {
+    display: flex;
+    position: absolute;
+    flex-direction: column;
+    height: 100vh;
+    width: 100%;
+    justify-content: start;
+    align-items: flex-start;
+    top: 48px;
+    left: 0;
+    background-color: #fff;
+    overflow-y: hidden;
+  }
+
+  .visible-mobilemenu-open {
+    display: flex;
+    flex-direction: column;
+  }
+
+  .visible-mobilemenu-opened {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+  }
+
+  .logo-link {
+    order: 2;
+    align-self: center;
+    margin-top: 35px;
+  }
+
+  .navbar-bottom-item {
+    text-align: center;
+    margin: 15px 0 0 0;
+  }
+
+  .navbar-bottom-item a {
+    text-align: center;
+    align-self: center;
+    margin: 0;
+  }
+
+  .last {
+    margin: 0;
+  }
+}
+
+@media screen and (min-width: 768px) {
+  .mobile-menu-button {
+    display: none;
+  }
+
+  .nav-top-open {
+    display: inline;
+  }
+
+  .mobile-menu-button-link {
+    display: inline;
+    position: fixed;
+    top: 20px;
+    left: 30px;
+    color: black;
+  }
+
+  .mobile-menu-button-link img {
+    width: 25px;
+  }
+
+  .headline-container {
+    background-image: url("images/header-bg-desktop.png");
+    background-repeat: no-repeat;
+    background-size: cover;
+  }
+
+  .headline-title h1 {
+    color: #c23f30;
+    font-size: 4vw;
```

Comment:
- [ ] Please endeavour to use the exact color scheme specified in the design. In this case, the right colour would be `#ec5242` 🙏

Reply:
Hello codepantha! Thank you for the requested change. I kindly changed the color for this element as per the design template.